nautobot_ssot/integrations/infoblox/utils/client.py

```diff
@@ -1224,7 +1224,8 @@ def update_ipaddress(self, ip_address, **data):  # pylint: disable=inconsistent-
         logger.info("Infoblox IP Address updated: %s", response.json())
         return response.json()
 
-    def get_network_containers(self):
+    # credit to @Eric-Jckson for this code: https://github.com/nautobot/nautobot-plugin-ssot-infoblox/pull/143
```

Comment:
Same comment as above

Reply:
Done as part of moving nested functions to client utils.